packages/jest-haste-map/package.json

```diff
@@ -34,7 +34,7 @@
     "slash": "^3.0.0"
   },
   "optionalDependencies": {
-    "fsevents": "^2.1.2"
+    "fsevents": "^2.2.1"
```

Comment:
Need to run `yarn` to update the lockfile

Reply:
Should (hopefully) be done!

Reply:
Seems like you used Yarn 1 to do that instead of the checked in version, how old is your Yarn 1 version since it didn't pick up on the `yarn-path` setting?

Reply:
@merceyz I tried to fix it, but the patch in yarn v2 seems broken.

```
➤ YN0066: │ fsevents@patch:fsevents@npm%3A2.3.1#builtin<compat/fsevents>::version=2.3.1&hash=127e8e: Cannot apply hunk #1
➤ YN0000: │     }
➤ YN0000: │
➤ YN0028: │ -   let instance = Native.start(path, handler);
➤ YN0028: │ +   let VFS = require('./vfs');
➤ YN0028: │ +   let vfs = new VFS(path);
➤ YN0028: │ +   let instance = Native.start(vfs.resolvedPath, vfs.wrap(handler));
➤ YN0000: │     if (!instance) throw new Error(`could not watch: ${path}`);
➤ YN0000: │     return () => {
```

Reply:
That was fixed in https://github.com/yarnpkg/berry/pull/2078

Reply:
Oh, the fork was super outdated and had an old version of yarn